core/azure-core-mgmt/src/main/java/com/azure/core/mgmt/AsyncOperationResource.java

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.common.mgmt;
+package com.azure.core.mgmt;
```

Comment:
As @conniey has stated, and as I discussed in the standup today, `mgmt` should be `management`, as outlined in #3349, and it should be done as part of this PR ideally (to lessen the pain to one PR).

Reply:
Done with the renaming mgmt. to management for azure core.